README.md

```diff
@@ -129,3 +129,18 @@ go mod tidy
 ```
 
 Then commit the changes to `go.mod` and `go.sum`.
+
+### Generating Kibana clients
+
+Kibana clients for some APIs are generated based on Kibana OpenAPI specs.
+Please see [Makefile](./Makefile) tasks for more details.
+
+#### Known issues
+OpenAPI generator produces incorrect code for `RunConnector200ResponseData` struct
+in [model_run_connector_200_response_data.go](./generated/kibanaactions/model_run_connector_200_response_data.go).
+It doesn't strip curly braces from the struct field names.
+
+[There is a fix for the issue](https://github.com/OpenAPITools/openapi-generator/pull/13167) but it's not yet merged.
+Currently the file is fixed manually.
```

Comment:
Is it possible to detail the manual fixes applied to the file, perhaps in `generated/connectors/README.md`?

Reply:
Added `generated/connectors/README.md`.